### ISSUE TITLE
Fix ProcessTest.test_sanity

### ISSUE THREAD

**Issue by Smjert** [osquery/osquery#6358]
```
73: [----------] 1 test from ProcessesTest
73: [ RUN      ] ProcessesTest.test_sanity
73: E0402 18:27:22.879258  9708 processes.cpp:312] Failed to lookup path information for process 4
73: E0402 18:27:22.879258  9708 processes.cpp:332] Failed to get cwd for 4 with 31
73: D:\a\1\s\tests\integration\tables\helper.cpp(181): error: Value of: boost::get<CustomCheckerType>(validator)(value)
73:   Actual: false
73: Expected: true
73: Custom validator of the column "start_time" with value "1585109187" failed
73: Row: {cmdline: "", cwd: "", disk_bytes_read: "202992268", disk_bytes_written: "78292959", egid: "-1", elapsed_time: "742855", euid: "-1", gid: "-1", handle_count: "1028", is_elevated_token: "-1", name: "System", nice: "32", on_disk: "-1", parent: "0", path: "", percent_processor_time: "2437343750", pgroup: "-1", pid: "4", resident_size: "147456", root: "", sgid: "-1", start_time: "1585109187", state: "STILL_ACTIVE", suid: "-1", system_time: "243734", threads: "119", total_size: "126976", uid: "-1", user_time: "0", wired_size: "0"}
73: D:\a\1\s\tests\integration\tables\helper.cpp(181): error: Value of: boost::get<CustomCheckerType>(validator)(value)
73:   Actual: false
73: Expected: true
73: Custom validator of the column "start_time" with value "1585109187" failed
73: Row: {cmdline: "\\SystemRoot\\System32\\smss.exe", cwd: "C:\\Windows\\System32\\smss.exe", disk_bytes_read: "29233", disk_bytes_written: "33", egid: "-1", elapsed_time: "742855", euid: "-1", gid: "18", handle_count: "51", is_elevated_token: "1", name: "smss.exe", nice: "32", on_disk: "1", parent: "4", path: "C:\\Windows\\System32\\smss.exe", percent_processor_time: "1406250", pgroup: "-1", pid: "356", resident_size: "1257472", root: "C:\\Windows\\System32\\smss.exe", sgid: "-1", start_time: "1585109187", state: "STILL_ACTIVE", suid: "-1", system_time: "140", threads: "2", total_size: "405504", uid: "18", user_time: "0", wired_size: "2536"}
73: D:\a\1\s\tests\integration\tables\helper.cpp(181): error: Value of: boost::get<CustomCheckerType>(validator)(value)
73:   Actual: false
73: Expected: true
[...]
```

**Comment by Smjert** [comment]
I tried to give this a go, it seems that the issue is either the boot time calculated by the test is wrong and it's calculated with  `now - getUptime()`, or the start_time is.

The start_time strangely results to be 10h earlier than the boot_time, and the boot_time is very near to when the actual build started (and so the VM started to do its CI job).

The API used to get the start_time is `GetProcessTimes`, which I don't think should be influenced by anything, but the boot_time is taken via `GetTickCount64`, which I might imagine that it could be reset in some situations when using a VM.
Although when checking for ways to obtain an uptime and or boot time on Windows, I've only found that, and other two methods, WMI and EventLog, which do not seem reliable.
